lib/elixir/lib/kernel.ex

```diff
@@ -912,8 +912,10 @@ defmodule Kernel do
   @doc """
   Concatenates two lists.
 
-  The complexity of `a ++ b` is proportional to the `length(a)`, so avoid appending
-  to lists of arbitrary length. Instead, prepend with `[item | rest]` and reverse.
+  The complexity of `a ++ b` is proportional to `length(a)`, so avoid appending
+  to lists of arbitrary length ie: `list ++ [item]`
```

Comment:
Maybe say `prepending via `[item | rest]` recursively and then reversing at the end.`? :confused: